conf.d/cassandra.yaml.example

```diff
@@ -66,6 +66,14 @@ init_config:
           - MemtableLiveDataSize
           - MemtableSwitchCount
           - MinRowSize
+          - ReadLatency
+          - WriteLatency
+        attribute:
+          - Value
```

Comment:
`Value` did not return any metrics in my tests (Cassandra `2.1.13`). @sethrosenblum could you confirm that it is valid please?